internal/test/t1/Makefile

```diff
@@ -1,2 +1,2 @@
 regenerate:
-	gocc t1.bnf
\ No newline at end of file
+	gocc -p github.com/goccmack/gocc/internal/test/t1 t1.bnf
```

Comment:
All looks great to me, is the `-p`s really necessary or can we set the GOPATH as part of the CI setup?

Reply:
I had multiple issues where gocc did put parts of my the user home path as include path if I did not force it by using -p, so I put it here by default.  You can see an example in this failed CI run: https://github.com/cornelk/gocc/runs/1793013769

```
git diff .
diff --git a/example/astx/errors/errors.go b/example/astx/errors/errors.go
index 640214b..666ce02 100755
--- a/example/astx/errors/errors.go
+++ b/example/astx/errors/errors.go
@@ -6,7 +6,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/goccmack/gocc/example/astx/token"
+	"home/runner/work/gocc/gocc/example/astx/token"
 )

```

Reply:
Are you sure setting the GOPATH does not help?

Reply:
This is the logic I think of how the default out dir is chosen https://github.com/goccmack/gocc/blob/47605ef073d33e0a47a9209d61fa0eff182a95c6/internal/config/config.go#L204